### ISSUE TITLE
Plugin configuration requests blocked by AWS firewall rule RestrictedExtensions_QUERYARGUMENTS

### ISSUE THREAD

**Issue by kdckrs** [shopware/shopware#1562]
**Description**  
On environments that are shielded by the Core rule set (CRS) of the AWS Firewall you do not have the ability to view the config settings of a module. The following requests are blocked:

- https://example.com/api/v2/_action/system-config?domain=somePluginName.config
- https://example.com/api/v2/_action/system-config/schema?domain=somePluginName.config
- https://example.com/api/v2/_action/system-config/check?domain=somePluginName.config

The rule that is blocking these requests:
RestrictedExtensions_QUERYARGUMENTS: Inspects requests whose query arguments are system file extensions that the clients shouldn't read or run. Example patterns include extensions like .log and .ini. 

Taken from [https://docs.aws.amazon.com/waf/latest/developerguide/aws-managed-rule-groups-list.html](https://docs.aws.amazon.com/waf/latest/developerguide/aws-managed-rule-groups-list.html)

**Possible Solution**  
- Modify/rewrite the path of the API calls so that it's no longer triggered

**Comment by MarcelSchmaeing** [comment]
Hello,

Within our company roadmap and work capacity, we try to address each bug or improvement request but admit that not each one will be resolved. To continue our culture of honesty and openness, we are closing this issue to focus on our roadmap on behalf of all Shopware users.

If you feel like this issue is still impacting you, please create a new issue and let us know.
Thank you again for your valuable feedback.
